RTL_testcases/RTL_Benchmarks_Gap_Analysis/leon3mp/raptor_sdc.sdc

```diff
@@ -1,3 +1,3 @@
-create_clock -period 2.5 clk
+create_clock -period 2.5 -name clk
```

Comment:
This is incorrect, with the -name option you are creating a virtual clock. clk is a real clock.